### ISSUE TITLE
Map does not render on Tor Browser

### ISSUE THREAD

**Issue by contrapunctus-1** [StrandedKitty/streets-gl#181]
**Describe the bug**
1. Open https://streets.gl/ in Tor Browser.
2. The UI is visible, but the map is not.

**Screenshots**
![Screenshot from 2024-03-21 01-28-14](https://github.com/StrandedKitty/streets-gl/assets/8513118/b25c4aa0-a965-4303-bb1f-bb5cec3b87d3)

**System information**
 - OS: Linux 6.6.15-amd64 #​1 SMP PREEMPT_DYNAMIC Debian 6.6.15-2 (2024-02-04) x86_64 GNU/Linux
 - Browser with version: Tor Browser 13.0.12 (based on Mozilla Firefox 115.9.0esr)

Here's a log from the inspector.
[streets.gl_Archive [24-03-21 01-12-43].har.zip](https://github.com/StrandedKitty/streets-gl/files/14672000/streets.gl_Archive.24-03-21.01-12-43.har.zip)

**Comment by TimGoodwyn** [comment]
I get the same behaviour on plain Firefox 123.0.1 on Mac OS 12.6.5

**Comment by applemayexist** [comment]
I can reproduce on Tor Browser 13.0.13, but I would think that this is a feature, not a bug. WebGL can be used to fingerprint your device, and so Tor is likely disabling it.

**Comment by contrapunctus-1** [comment]
FWIW Tor Browser is able to render demo.f4map.com and maps.jawg.io (both of which have 3D buildings) just fine.

This looks relevant - https://gitlab.torproject.org/legacy/trac/-/issues/16005

**Comment by applemayexist** [comment]
You're right, it turns out WebGL is enabled, but only WebGL 1.0. Visiting about:support shows a large list of enabled WebGL 1.0 extensions, whereas WebGL 2.0 is empty. And sure enough, streets.gl [uses WebGL 2.0](https://github.com/StrandedKitty/streets-gl/tree/dev/src/lib/renderer/webgl2-renderer). Unfortunately I wasn't able to find anything about enabling WebGL 2.0 on Tor.
